src/main/java/org/jabref/logic/cleanup/MoveFilesCleanup.java

```diff
@@ -31,17 +31,21 @@
     private final BibDatabaseContext databaseContext;
     private final FileDirectoryPreferences fileDirectoryPreferences;
 
-    private final LayoutFormatterPreferences layoutPrefs;
+    // FIXME: remove unused variables later S.G.
+    // private final LayoutFormatterPreferences layoutPrefs;
     private final String fileDirPattern;
 
     private LinkedFile singleFileFieldCleanup;
 
+    // FIXME: remove the unused 'layoutPrefs' parameter later (S.G.); so far, however, it is left to
+    // prevent constructor signature change.
     public MoveFilesCleanup(BibDatabaseContext databaseContext, String fileDirPattern,
             FileDirectoryPreferences fileDirectoryPreferences, LayoutFormatterPreferences layoutPrefs) {
         this.databaseContext = Objects.requireNonNull(databaseContext);
         this.fileDirPattern = Objects.requireNonNull(fileDirPattern);
         this.fileDirectoryPreferences = Objects.requireNonNull(fileDirectoryPreferences);
-        this.layoutPrefs = Objects.requireNonNull(layoutPrefs);
+        // FIXME: Remove unused code later S.G.
```

Comment:
Please remove the commented-out code here.

Reply:
Fixed.